src/api/middleware.js

```diff
@@ -22,9 +22,15 @@ export function match(regexp: RegExp) {
   };
 }
 
-export function securityIframe(req: $RequestExtend, res: $ResponseExtend, next: $NextFunctionVer) {
+export function securityWebHeadersSecurity(req: $RequestExtend, res: $ResponseExtend, next: $NextFunctionVer) {
```

Comment:
security...Security seems a bit - too much security :D

Reply:
haha true ... I just want to be secure